examples/tutorials/get-started/flow-as-function.ipynb

```diff
@@ -85,8 +100,8 @@
     "f = load_flow(\n",
     "    source=flow_path,\n",
     ")\n",
-    "# need to create the connection\n",
-    "f.context.connections={\"classify_with_llm\": {\"connection\": \"new_ai_connection\"}}\n",
+    "# directly use connection created above\n",
+    "f.context.connections={\"classify_with_llm\": {\"connection\": conn}}\n",
```

Comment:
will created conn has secrets?

Reply:
If the connection object is built-in connection, they will have secrets.